frontend/src/components/SettingsConnections.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div class="flex flex-col gap-8">
+    <div class="text-3xl font-thin text-gray-500 dark:text-gray-200">{{ t('heading.connectedAccounts') }}</div>
+    <div class="max-w-3xl pl-6" v-for="(connection, category) in connections" v-bind:key="category">
+      <h2 class="mb-4 text-xl font-medium">{{ t(`heading.settings.connectedAccounts.${category}`) }}</h2>
+      <p>{{ t(`text.settings.connectedAccounts.connect.${category}`) }}</p>
+      <div v-if="category === 'google'" class="pt-2">
+        <p>
+          <i18n-t :keypath="`text.settings.connectedAccounts.connect.${category}Legal.text`" tag="label" :for="`text.settings.connectedAccounts.connect.${category}Legal.link`">
+          <a class="underline" href="https://developers.google.com/terms/api-services-user-data-policy" target="_blank">{{ t(`text.settings.connectedAccounts.connect.${category}Legal.link`) }}</a>
+          </i18n-t>
```

Comment:
suggestion (non-blocking): Some indentation here for better readability:

```suggestion
          <i18n-t
            :keypath="`text.settings.connectedAccounts.connect.${category}Legal.text`"
            tag="label"
            :for="`text.settings.connectedAccounts.connect.${category}Legal.link`"
          >
            <a class="underline" href="https://developers.google.com/terms/api-services-user-data-policy" target="_blank">
              {{ t(`text.settings.connectedAccounts.connect.${category}Legal.link`) }}
            </a>
          </i18n-t>
```